checkstyle/suppressions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+"-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+"https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+    <suppress id="core" files="[/\\]src[/\\]main[/\\]java[/\\]br[/\\]com[/\\]sbk[/\\]sbking[/\\]gui[/\\].*" />
```

Comment:
These seem strange as the regexp are "inverted". Anyway, it is correct.

Reply:
I just pushed a comment explaining why the tags are inverted :)